docs/core/connectors/mysql.md

```diff
@@ -53,4 +53,5 @@ See the [official documentation](https://dev.mysql.com/doc/refman/8.0/en/connect
 - `user`: The database user, e.g. `admin`.
 - `password`: The password for the database user.
 - `ssl`: Whether or not your database server uses SSL.
-- `connection_limit` (coming soon): The connection limit specifies the maximum number of simultaneous connections that Prisma might have open to your database. **Default**: `1`.
+- `connection_limit`: The connection limit specifies the maximum number of simultaneous connections that Prisma might have open to your database.
+- `socket`: If you want to connect via sockets, this parameter must specified a file path to the socket you want to use, e.g.: `/var/run/mysqld/mysqld.sock`. (Learn more in this [GitHub issue](https://github.com/prisma/prisma2/issues/525)).
```

Comment:
We do not really want the user to read all 33, partially hidden, comments of this thread, right? If there is one specific comment that contains the information we should link to it. If this is not the case, we should create this location in a new issue or include it here in the docs directly.